### ISSUE TITLE
Aboult the warmup_n_steps and accum_grad_n_steps usd in Aishell 

### ISSUE THREAD

**Issue by mlxu995** [hirofumi0810/neural_sp#137]
In the config transformer.yaml, "n_epochs=30, batch_size=32, warmup_n_steps=25000, accum_grad_n_steps=8".
Under the above config, I got a learning rate curve which went straight up until the end of the training. 
And the result is not good as it reported in this project.
I need assistance, really appreciate. @hirofumi0810 @boji123

**Comment by hirofumi0810** [comment]
I have never met such a situation. Did you modify the config file?
